src/Map/index.js

```diff
@@ -63,49 +63,49 @@ export default class Map extends Component {
     zoom: 13,
     markerType: null,
     mapConfigLoaded: false,
-    loaded: false,
+    isLoaded: false,
+    isLoading: false,
   }
-  isBrowser = true
 
   componentDidMount() {
-    if (!this.props.editor) {
-      this.isBrowser = typeof document !== 'undefined'
```

Comment:
Why are we removing this check?

Reply:
The `isBrowser` property is not used anywhere. There are lot of things in this file that aren't used and should be cleaned up, I plan on doing that next.